src/components/Product.js

```diff
@@ -0,0 +1,25 @@
+import CartIcon from "../assets/svgs/cart.svg";
+import { useNavigate } from "react-router-dom";
+import { ROUTE_PATH } from "../constants/page";
+
+export default function Product({ product }) {
+  const navigate = useNavigate();
+
+  function handleCartClick() {
+    if (confirm("장바구니에 물건이 담겼습니다. 장바구니로 이동하시겠습니까?"))
+      navigate(ROUTE_PATH.CART);
+  }
+
+  return (
+    <div data-id={product.id}>
+      <img src={product.imageUrl} alt={product.name} />
+      <div className="flex justify-between w-280 p-5">
+        <div className="product-info">
+          <span className="product-info__name">{product.name}</span>
+          <span className="product-info__price">{product.price}원</span>
+        </div>
+        <img src={CartIcon} alt="장바구니" onClick={() => handleCartClick()} />
```

Comment:
(마이너) onClick에서 새로 익명함수를 생성하지않고 이렇게 전달 가능해보여요
```suggestion
        <img src={CartIcon} alt="장바구니" onClick={handleCartClick} />
```

Reply:
렌더링 시마다 익명함수를 호출해서 반복적으로 함수를 생성하고 있었네요 😅
넘겨줄 값이 없다면 함수 자체를 전달하도록 의식적으로 작성해보겠습니다 :)

[75cd5fe](https://github.com/next-step/react-shopping-cart/pull/29/commits/75cd5fe0273b39d953eb5acc200243666a4cce0f)